### PR TITLE
AMP-76919 Add Mailchimp streaming docs

### PR DESCRIPTION
**Pull request by daniel-beer**
# Amplitude Developer Docs PR

## Description

Add docs for Mailchimp streaming. Clean up linter errors in HubSpot docs.

## Deadline

End of week

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/hubspot-event-streaming.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ Keep these things in mind when sending events to Hubspot:
 5. Enter the HubSpot Event Name created by HubSpot
 6. Toggle the **Send events** filter to select the events to send. HubSpot recommends choosing the events that are most important to your use case. 
 7. Click on the **Select & filter events** option to choose the event type of the events you would like to send.
-8. Click on **Map properties to destination** to specify the identity mapping between Amplitude and HubSpot. Please note that HubSpot requires the identifier to be at least one of the following: utk, email, or object ID.
+8. Click on **Map properties to destination** to specify the identity mapping between Amplitude and HubSpot. Note that HubSpot requires the identifier to be at least one of the following: utk, email, or object ID.
```
Comment (sina-jamshidi) [inline docs/data/destinations/hubspot-event-streaming.md:46]: perhaps in unordered bullets?

**Review by sina-jamshidi — APPROVED** [review]

File: docs/data/destinations/mailchimp-event-streaming.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        ![Setting up a filter for anonymous users on events](/../assets/images/streaming-anonymous-users-filter.png)
+
+2. (optional) In **Select additional properties**, select any more event and user properties you want to send to Mailchimp. If you don't select any properties here, Amplitude doesn't send any. These properties are sent to Mailchimp as [Mailchimp event properties](https://mailchimp.com/developer/marketing/api/list-member-events/add-event/). _Transformed event properties and transformed user properties aren't supported._
+
+### Configure user forwarding
+
+Under **Send Users**, make sure the toggle is enabled ("Users are sent to Mailchimp") if you want to stream users and their properties to Mailchimp. When enabled, users are automatically created or updated in Mailchimp when an event is sent to Amplitude. [Amplitude Identify API](https://www.docs.developers.amplitude.com/analytics/apis/identify-api/) calls are also forwarded to Mailchimp. Users aren't sent on a schedule or on-demand using this integration.
```
Comment (bgiori) [inline docs/data/destinations/mailchimp-event-streaming.md:51]: Use internal links please! like `/analytics/apis/identify-api.md`

**Review by bgiori — APPROVED** [review]
Approved provided a quick link change

File: docs/data/destinations/mailchimp-event-streaming.md (hunk 2 above)
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/mailchimp-event-streaming.md:51]: @daniel-beer  This one says "Braze"

Amplitude CDP's Braze streaming integration enables you to forward your Amplitude events and users straight to Mailchimp with just a few clicks.

Comment (daniel-beer) [reply docs/data/destinations/mailchimp-event-streaming.md]: Good catch @Brandoncharleskhoo1. In the future, it helps if you make a separate comment on the specific line. That makes it much quicker for me to find and allows me to resolve other comments (such as the one above) one at a time as I address them.

**Pull request merged**